spec/abi.dd

```diff
@@ -9,16 +9,15 @@ $(SPEC_S Application Binary Interface,
 	D binaries built by other implementations.
 	)
 
-$(SECTION3 C ABI,
+$(H3 $(LNAME2 c_abi, C ABI))
```

Comment:
personal taste, SECTION<X> tend to lead to right paren error since the content is surrounded, which is not the case when using H<X>.

But I've checked the output on your site and it looks OK.